src/http/server.rs

```diff
@@ -57,6 +61,22 @@ async fn entity_validator(
     }
 }
 
+fn build_cors(ayb_cors: Option<AybConfigCors>) -> Cors {
+    let mut cors = Cors::default().allow_any_header().allow_any_method();
+
+    if ayb_cors
+        .as_ref()
+        .is_some_and(|conf| conf.origin.trim() == "*")
```

Comment:
Brainstorming alternatives --- we could make the CORS configuration required, and default it to `*`, which would simplify some of this logic.

Reply:
I don't feel too strongly

Reply:
I think it should be required, that way the instance owner is aware that CORS is set to `*` by default without having to read documentation. I'll change it